roles/manage-aws-infra/tasks/update_dns.yml

```diff
@@ -12,7 +12,7 @@
     aws_access_key: "{{ aws_access_key }}"
     aws_secret_key: "{{ aws_secret_key }}"
     zone: "{{ public_dns_domain }}."
-    record: "master.{{ env_id }}.{{ public_dns_domain }}."
+    record: "master-0.{{ env_id }}.{{ public_dns_domain }}."
```

Comment:
this shouldn't be hardcoded to `master-0`, but should be driven by the inventory/instances created

Reply:
This is actually driven by the inventory in terms or dynamically getting the elastic IP assigned to the master. 'master-0' is just an "alias" in this case to be used externally to resolve that IP.

```- name: Create elastic IP for Master Node (Non HA Mode)
  ec2_eip:
    aws_access_key: "{{ aws_access_key }}"
    aws_secret_key: "{{ aws_secret_key }}"
    region: "{{ aws_region }}"
    in_vpc: yes
    state: present
    release_on_disassociation: yes
    device_id: "{{ instances_master_etcd.results[0].instances[0].id }}"
  when: not ha_mode
  register: master_eip